app/models/fs/box.rb

```diff
@@ -40,6 +40,10 @@ def self.create_with_api_connection!(params)
   def sync
   end
 
+  def single_recipient?
+    false
```

Comment:
nie je toto true?

Reply:
pravda, neotocila som to

Reply:
Ale preco testy nepadli?

Reply:
lebo som tu logiku neotocila na oboch miestach, logika zostala rovnaka, iba som premenovala metodu, takze prakticky sa nic nepokazilo, len biznisovo to zmysel nedavalo.